docs/version-7-upgrade.md

```diff
@@ -169,25 +169,15 @@ const provider = new aws.Provider("provider", {
 
 ## S3 Bucket/BucketV2 Changes
 
-As part of this major version upgrade we are removing the old S3 `Bucket` resource
-and are renaming `BucketV2` to `Bucket`. This change should simplify things for
-users and make it easier to understand which bucket resources to use. There will
-now only be one version of bucket resources without any version information.
-
-As part of this change there will be breaking changes in the Bucket SDKs
-that fall into two categories.
-
-1. Renamed resources. Users will need to update their code to rename the
-   resource to the new name (e.g. `BucketV2` => `Bucket`)
-2. MaxItemsOne changes. Fields that were previously (in v6) lists that only
-   accepted a single item have been changed to no longer be lists. For example,
-   the `serverSideEncryptionConfigurations` field on `BucketV2` was a list that
-   only accepted a single configuration item. This is now
-   `serverSideEncryptionConfiguration`.
- 
-### Renamed Resources
-
-- `aws:s3/bucketV2:BucketV2` => `s3/bucket:Bucket`
+In `v6` of the Pulumi AWS Provider the `s3.Bucket` and `s3.BucketV2` resources represent different resource implementations. `s3.BucketV2` represents the latest version of the upstream Terraform resources, while `s3.Bucket` is a separate resource maintained by Pulumi to keep backwards compatibility with the `v4` release of the upstream Terraform Provider.
+
+In `v7` we are taking the first step in unifying these two resources. In `v7` both `s3.Bucket` and `s3.BucketV2` will represent the latest version of the upstream Terraform S3 Bucket resource. As part of moving the `s3.Bucket` resource to the latest upstream implementation, there are a couple of SDK level breaking changes.
+
+- The `loggings` input property has been renamed to `logging` and the type has changed from a list to a single object.
+- The `website.routingRules` now only accepts a `string` input property.
+
+We have also introduced new Bucket configuration resources that are alternatives to their `V2` counterparts. The `V2` versions will be removed in `v8` of the Pulumi AWS Provider.
+
```

Comment:
We may want to add a recommendation for which bucket resource to go with? E.g. "if you're currently using `BucketV2` then do nothing but be advised it will become `Bucket` next year" and "if you're still on `Bucket` then this upgrade will be a little more involved, but after that you're Done and set for v$next-major"